### DISCUSSION TITLE
Anyone else having issues with render props types not showing?

### DISCUSSION THREAD

**Discussion by wking-io** [tailwindlabs/headlessui#1616] [Help]
Using the latest version of headlessui/react with typescript we are now getting errors or render prop types. Is there an issue with configuration on my side or is anyone else experiencing this issue?

**Comment by Sowed** [comment]
Yeah, We are also experience the same issue. Running the shown [listbox example](https://headlessui.com/react/listbox), or any other one that uses render props. We get lots of typescript error.

Realised the `Listbox` `children` prop is typed as `children?: React.ReactNode` yet it the example uses it as a render prop. Currently using version `1.7.3`.

Showing `'({ open }: ListboxRenderPropArg<SetStateAction<MyListItemsItemType>>) => Element' is not assignable to type 'string'`.

I suspect the issue might be coming with the render prop function type and string.
